app/full_text_search.py

```diff
@@ -241,15 +281,17 @@ def create_romanized_name_fields(romanize_method, **kwargs):
     fields = []
     romanized_names_list = []
     romanize_method_name = romanize_method.__name__
-    for field_name in kwargs:
-        romanized_names = romanize_method(kwargs[field_name])
+    for field_name, field_value in kwargs.iteritems():
+        romanized_names = romanize_method(field_value)
         for index, romanized_name in enumerate(romanized_names):
-            fields.extend(create_fields_for_rank('%s_%d' %
-                                                 (field_name, index),
+            fields.extend(create_fields_for_rank('%s_%s_%d' %
+                                                 (field_name,
+                                                  romanize_method_name, index),
                                                  romanized_name))
         romanized_names_list.extend(romanized_names)
 
-    full_name_fields, romanized_full_names = create_full_name_without_space_fields(
+    full_name_fields, romanized_full_names =\
```

Comment:
Insert white space between "=" and "\". Or put "(" instead here and close it at the end of the statement.